release-doc-in-process/all_vf.TTL

```diff
@@ -346,9 +366,74 @@ vf:fulfills     a owl:ObjectProperty ;
 
-# isSubstitutable "Defines if any resources of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc."
+vf:isSubstitutable a    owl:DatatypeProperty ;
+        rdfs:label      "is substitutable" ;
+        rdfs:domain     vf:ResourceType ;
+        rdfs:range      xsd:boolean ;
+        rdfs:comment    "Defines if any resource of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc." .
+
+#vf:unit a               owl:ObjectProperty ;
+#        rdfs:label      "unit" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:range      qudt:unit ;
+#        rdfs:comment    "The unit expected for this resource type as a default." .
+
+vf:definedQuantity a    owl:ObjectProperty ;
+        rdfs:label      "defined quantity" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The amount and unit defined for this recipe flow." .
+
+# TODO: need to define this, something like currency, inventory, work, service; may be other aspects too.
+#vf:resourceCategory a    owl:ObjectProperty ;
+#        rdfs:label      "category" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:comment    "Defines the category of behavior expected for the type of resource." .
+
+vf:developedBy a        owl:ObjectProperty ;
+        rdfs:label      "developed by" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      foaf:Agent ;
+        rdfs:comment    "The agent who developed this recipe." .
+
+vf:variantOf a           owl:ObjectProperty ;
+        rdfs:label      "variant of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is a variation of another recipe." .
+
+vf:successorOf a        owl:ObjectProperty ;
+        rdfs:label      "successor of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is an updated version of another recipe." .
+
+vf:actsOn a             owl:ObjectProperty ;
+        rdfs:label      "acts on" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      vf:ResourceType ;
+        rdfs:comment    "Defines the type of resource this recipe flow acts on." .
+
+vf:hasDuration a        owl:ObjectProperty ;
+        rdfs:label      "duration" ;
+        rdfs:domain     vf:ProcessType ;
+        rdfs:range      time:TemporalDuration ;
+        rdfs:comment    "The calendar duration defined for this type of process." .
+
+vf:resourceTypedAs
+        a                   owl:ObjectProperty ;
+        rdfs:label          "resource typed as" ;
+        rdfs:domain         [ owl:unionOf (vf:Commitment vf:EconomicResource) ] ;
+        rdfs:range          vf:ResourceType ;
+        rdfs:comment        "A resource's primary classification, part of describing that resource." .
+
+vf:processTypedAs
```

Comment:
I assume that process can use `vf:classifiedAs` to reference taxonomy and this only would reference a `vf:RecipeProcess` from whjich it was created. I would prefer something more specific as `vf:basedOn`. We may also need to use it to relate flows to `vf:RecipeFlow` from which they were created.

Reply:
basedOn is good, thanks!

Reply:
>We may also need to use it to relate flows to vf:RecipeFlow from which they were created.

Not that I know of. Let's wait and see if that is needed for anything.